tests/db_engine_specs_test.py

```diff
@@ -201,3 +201,49 @@ def test_get_datatype(self):
         self.assertEquals('TINY', MySQLEngineSpec.get_datatype(1))
         self.assertEquals('VARCHAR', MySQLEngineSpec.get_datatype(15))
         self.assertEquals('VARCHAR', BaseEngineSpec.get_datatype('VARCHAR'))
+
+    def test_limit_with_implicit_offset(self):
+        self.sql_limit_regex(
+            textwrap.dedent("""\
+                SELECT
+                    'LIMIT 777' AS a
+                    , b
+                FROM
+                table
+                LIMIT
+                99990, 999999"""),
+            textwrap.dedent("""\
+            SELECT
+                'LIMIT 777' AS a
+                , b
+            FROM
+            table LIMIT 1000, 999999"""),
```

Comment:
Note I believe the form here is `LIMIT [<offset>,] <row-count>` per [this](https://dev.mysql.com/doc/refman/8.0/en/select.html) and it seems this is flipped.